test/notBoth.js

```diff
@@ -5,7 +5,20 @@ import * as RA from '../src/index';
 import eq from './shared/eq';
 
 
+const supportsFantasyLand = () => {
```

Comment:
Seems a little bit cryptic. Can we simplify to following ?

```js
const supportsFantasyLand = () => {
  try {
    return RA.notBoth(Just(true), Just(true)).equals(Just(false));
  } catch (e) {
    return false;
  }
}

Reply:
Agreed.